_topics/en/simplicity.md

```diff
@@ -10,14 +10,31 @@ categories:
 ## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
 excerpt: >
   **Simplicity** is a work in progress low level programming language designed
-  to offer greater functionality than Bitcoin Script. It will require a future
-  soft fork to be enabled on Bitcoin.
+  to offer greater flexibility and expressiveness than Bitcoin Script whilst
+  allowing you to verify the safety, security and costs of your program. It
+  offers native Merklized scripting, formal semantics and type checking.
+  It is not Turing complete but it can verify the execution of Turing complete
+  programs. Although Simplicity provides certain proofs of correctness,
```

Comment:
My guess is that you're trying to say that solutions generated by a turing complete system don't necessary need to be evaluated by a turing complete system in order to be proven correct.  For that, you verify *results* not *execution*.  Think of verifying results as your boss inspecting something you built and verifying execution as your boss watching you while you work.  The former can be effectively done with a static checklist; the later can only be effectively done by someone who can understand human dynamism.

Verifying the results of turing complete systems is something that Bitcoin Script can already do, so I don't think it's a particularly noteworthy feature (though I'm sure it's easier given the increased expressiveness of Simplicity).  I would personally suggest dropping the whole sentence about turing completeness, but if you want to keep it, I think s/execution/results/ would be sufficient.

Reply:
This is a really good analogy, thanks. But are there three options here for a "turing complete" program? All the nodes on the network actually execute the program to verify the execution (Ethereum), no nodes execute the program, they just verify the result meets a limited set of rules (Bitcoin Script) and a third: no one runs the program but they verify the execution was done correctly without actually doing the execution themselves (potentially Simplicity?).

I'm in two minds on whether to keep it in or not. I think it is a common question (because of Ethereum) so I'm tempted to leave it in but if you think it is better removed I'll remove.

Reply:
I'll remove it

Reply:
> no nodes execute the program, they just verify the result meets a limited set of rules (Bitcoin Script) and a third: no one runs the program but they verify the execution was done correctly without actually doing the execution themselves (potentially Simplicity?)

Verification with Simplicity is the same as verification with Script, it's just that Simplicity offers a more expressive language.  It's just the difference between writing a program in one language versus another.